### ISSUE TITLE
RuntimeError: cannot find builtin font with name 'Arial'

### ISSUE THREAD

**Issue by aeilot** [ArtifexSoftware/pdf2docx#216]
I am trying to convert a PDF file to a Word file when the error goes: `RuntimeError: cannot find builtin font with name 'Arial'`. I am sure I have had the font `Arial` on my computer since it's the builtin font.

I guess this must have something to do with `PyMuPDF`. So I searched and saw [this](https://github.com/pymupdf/PyMuPDF/issues/2561). My PDF file is generated by printing a website to PDF, so I tried to change the font of my website to Helvetica so as to alter the font of the PDF file, yet the error info stayed the same. Pretty confusing.

My code's here:
```py
for i in range(len(download_list)):
    pdf_file = "%d.pdf" % i
    docx_file = "%d.docx" % i
    cv = Converter(pdf_file)
    cv.convert(docx_file)
    cv.close()
``` 
which is exactly what I'm told to do in the tutorial.

**Comment by su-heyang** [comment]
I followed this answer and changed the DEFAULT_FONT_NAME variable in constants.py, and my code is worked. 
 [](https://github.com/pymupdf/PyMuPDF/issues/2561#issuecomment-1655391691)
![DEFAULT_FONT_NAME](https://github.com/dothinking/pdf2docx/assets/85557188/660b1901-4e11-48b1-9084-f4e55e115e0e)

**Comment by su-heyang** [comment]
> 

https://github.com/pymupdf/PyMuPDF/issues/2561#issuecomment-1655391691

**Comment by su-heyang** [comment]
> 

https://github.com/pymupdf/PyMuPDF/issues/2561#issuecomment-1655391691

**Comment by JarvanBest** [comment]
Good idea,

**Comment by sneko** [comment]
Does any of you have a solution without modifying the library itself?

Thank you,

**Comment by dothinking** [comment]
> I followed this answer and changed the DEFAULT_FONT_NAME variable in constants.py, and my code is worked. ![DEFAULT_FONT_NAME](https://private-user-images.githubusercontent.com/85557188/261518999-660b1901-4e11-48b1-9084-f4e55e115e0e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzMDIzMDksIm5iZiI6MTcwNDMwMjAwOSwicGF0aCI6Ii84NTU1NzE4OC8yNjE1MTg5OTktNjYwYjE5MDEtNGUxMS00OGIxLTkwODQtZjRlNTVlMTE1ZTBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAzVDE3MTMyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1ZTc2ZmJiOTdjNzY2YzZmNmI4MTAyYWJkMjRiZWRkOTU4MGU2ZDg5MjAxOGI2YTA3ZjBiZTM0NWU1YTE4NWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dySqcz8GRhUr5ZCDmS0UKZ8nbF_EJFCQRfK-hMYsF0o)

This should be the right fix.

`DEFAULT_FONT_NAME` works when an unnamed font is used in PDF -> `pdf2docx` use this default font to update text span bounding box from pdf side (in case issues like [PyMuPDF#642](https://github.com/pymupdf/PyMuPDF/issues/642)).

`Arial` is a common font, but not 100% supported by any system. `helv` is a good option, as it is one of the base-14 fonts supported by any pdf. In addition, after conversion, Word processing software like MS Word is able to deal with `helv` as well.

**Comment by dothinking** [comment]
> Does any of you have a solution without modifying the library itself?
> 
> Thank you,

Fixing source code like above should be the easiest way before publishing a bug-fixing version.

**Comment by dothinking** [comment]
Fixed in version `v0.5.7`.
